src/js/media/media.js

```diff
@@ -239,7 +256,8 @@ vjs.MediaTechController.prototype.stopTrackingCurrentTime = function(){
   this.player().trigger('timeupdate');
 };
 
-vjs.MediaTechController.prototype.dispose = function() {
+vjs.MediaTechController.prototype.dispose = function(){
+  this.removeControlsListeners();
```

Comment:
This actuallyl shouldn't be needed. When you use `this.off(player, 'controlsenabled', this.activateControls);` it should automatically remove the activateControls listener from the player when this tech is disposed. It's [a feature of the off method](https://github.com/videojs/video.js/blob/master/src/js/component.js#L562).